test/llvm/debugInfo/dwarfdump/tuples.good

```diff
@@ -218,21 +219,21 @@ DW_TAG_structure_type
                 DW_AT_decl_file	("$CHPL_HOME/modules/internal/ChapelTuple.chpl")
                 DW_AT_decl_line	(26)
                 DW_AT_alignment	(1)
-                DW_AT_data_member_location	(0x38)
+                DW_AT_data_member_location	(0x18)
   DW_TAG_member
                 DW_AT_name	("x4")
-                DW_AT_type	("(int(64),real(64),string)")
+                DW_AT_type	("_ref((int(64),real(64),string))")
```

Comment:
confirming: this is because `fReturnByRef` has been switched off?

For my education, how is this a behavior-preserving optimization?

Reply:
No, this is a result of `--no-inline`